cleanlab/segmentation/summary.py

```diff
@@ -0,0 +1,332 @@
+            axes[plot_index].imshow(cmap[np.argmax(pred_probs[i], axis=0)])
+            axes[plot_index].set_title("Argmaxed Prediction Probabilities")
+            plot_index+=1
+        
+        # Third image - Errors
+
+        if output_plots == 1:
+            ax = axes
+        else:
+            ax = axes[plot_index]
+
+        mask = np.full((h,w), True) if len(exclude)== 0 else ~np.isin(labels[i], exclude)
+        ax.imshow(issues[i]& mask, cmap='gray', vmin=0, vmax=1)
+        ax.set_title(f"Suggested Errors in image index {i}")
+        plt.show()
+        
+        plot_index = 0
+
+    return None
+
+
+
+def common_label_issues(
+    issues: np.ndarray,
+    labels: np.ndarray,
+    pred_probs: np.ndarray,
+    *,
+    exclude: List[int] = [],
+    top: int = 20,
+    class_names: Optional[List[str]] = None,
+    verbose: bool = True
+) -> pd.DataFrame:
+    """
+    Display the frequency of which label are swapped in the dataset.
+
+    These may correspond to pixels that are ambiguous or systematically misunderstood by the data annotators.
+
+    Parameters
+    ----------
+    issues:
+        Boolean **mask** for the entire dataset
+        where ``True`` represents a pixel label issue and ``False`` represents an example that is
+        accurately labeled.
+
+        Same format as output by :py:func:`segmentation.filter.find_label_issues <cleanlab.segmentation.filter.find_label_issues>`
+        or :py:func:`segmentation.rank.issues_from_scores <cleanlab.segmentation.rank.issues_from_scores>`.
+
+    labels : np.ndarray 
+      Discrete array of noisy labels for a classification dataset, i.e. some labels may be erroneous.
+      *Format requirements*: for dataset with K classes, each pixel must be integer in 0, 1, ..., K-1.
+      For a standard (multi-class) classification dataset where each example is labeled with one class,
+      `labels` should be 3-D array of shape ``(N,H,W,)``. 
+      
+    Tip: If your labels are one hot encoded you can `np.argmax(labels_one_hot,axis=1)` assuming that `labels_one_hot` is of dimension (N,K,H,W)
+    before entering in the function
+
+    pred_probs : np.ndarray
+      Array of shape ``(N,K,H,W,)`` of model-predicted class probabilities,
+      ``P(label=k|x)``. Each pixel contains an array of K classes, where for 
+      an example `x` the array at each pixel contains the model-predicted probabilities 
+      that `x` belongs to each of the K classes.
+
+    class_names:
+        Optional length K list of names of each class, such that `class_names[i]` is the string name of the class corresponding to `labels` with value `i`.
+
+        If `class_names` is provided, display these string names for predicted and given labels, otherwise display the integer index of classes.
+
+    top:
+        Maximum number of tokens to print information for.
+
+    exclude:
+        Optional list of label classes that can be ignored in the errors, each element must be 0, 1, ..., K-1
```

Comment:
Does each element have to be in [0, 1, ..., K-1] or do they have to be ordered like 0, 1, ... K-1?